lib/src/controls/navigation/navigation_view/pane.dart

```diff
@@ -779,13 +782,15 @@ class _OpenNavigationPane extends StatefulWidget {
   _OpenNavigationPane({
     required this.pane,
     required this.theme,
+    this.shouldDrawHeaderSpaceIfEmpty = true,
     this.paneKey,
     this.listKey,
     this.onToggle,
     this.onItemSelected,
   }) : super(key: pane.key);
 
   final NavigationPane pane;
+  final bool shouldDrawHeaderSpaceIfEmpty;
```

Comment:
This property is used to know when the empty space is needed by another component like the `menuButton `when the `paneDisplayMode `is set to `open `or `minimal`. Also, I defaulted it to `true`, because there is only this two `paneDisplayMode `where the empty space is not used.

I haven't find another way to achieve this. That's why I've created this boolean.

Reply:
I believe it should be removed because, if empty, it shouldn't be rendered anyways. If the dev wants a space there, s/he can add the space by using a `SizedBox` on the header property. I believe this property is unecessary

Reply:
I Removed it! This work fine now without the property ;)